examples/angular-cli/src/stories/custom-metadata.stories.ts

```diff
@@ -1,4 +1,4 @@
-import { storiesOf } from '@storybook/angular';
+import { storiesOf, moduleMetadata } from '@storybook/angular';
```

Comment:
I think better having a separate example of this usage, also because it's necessary to check it in conjunction with addon-knobs.
How will it work with `template` instead of `component` prop?

Reply:
Added. Seems to work fine with `template`. I don't see any reason why they should affect each other, as this only touches the metadata, and passes other parts of the story through untouched.